src/components/atoms/AltinnAttachment.tsx

```diff
@@ -1,138 +1,49 @@
 import React from 'react';
 
-import { List, ListItem, ListItemIcon, ListItemText, makeStyles, Typography } from '@material-ui/core';
-import cn from 'classnames';
+import { Link, List } from '@digdir/design-system-react';
 
+import classes from 'src/components/atoms/AltinnAttachment.module.css';
 import { useLanguage } from 'src/hooks/useLanguage';
 import { FileExtensionIcon } from 'src/layout/FileUpload/FileUploadTable/AttachmentFileName';
 import { getFileEnding } from 'src/utils/attachment';
 import { makeUrlRelativeIfSameDomain } from 'src/utils/urls/urlHelper';
 import type { IDisplayAttachment } from 'src/types/shared';
-
-const useStyles = makeStyles(() => ({
-  a: {
-    margin: '4px 0',
-    borderRadius: 'var(--interactive_components-border_radius-normal)',
-    '&:hover': {
-      borderBottom: '0px',
-    },
-    '&:focus-within': {
-      borderBottom: '0px',
-      backgroundColor: 'transparent',
-      outline: 'var(--fds-focus-border-width) solid var(--fds-outer-focus-border-color)',
-      outlineOffset: 'var(--fds-focus-border-width)',
-      boxShadow: '0 0 0 var(--fds-focus-border-width) var(--fds-inner-focus-border-color)',
-    },
-    '&:active': {
-      borderBottom: '0px',
-    },
-    '&:after': {
-      display: 'none',
-    },
-  },
-  listItemPadding: {
-    paddingLeft: '2.0rem',
-  },
-  listItemPaddingNone: {
-    paddingLeft: '0rem',
-  },
-  listItemTextPadding: {
-    paddingLeft: '0',
-  },
-  inline: {
-    display: 'inline',
-  },
-  primaryText: {
-    fontWeight: 600,
-  },
-  icon: {
-    fontSize: '3rem',
-    color: 'var(--semantic-text-neutral-default)',
-    margin: '-8px 0',
-  },
-}));
-
 interface IAltinnAttachmentProps {
-  /** Attachments array with objects. See code example. */
   attachments?: IDisplayAttachment[];
-  /** Disables vertical padding (does not currently work in Styleguidist) */
-  listDisableVerticalPadding?: boolean;
-  /** Adds 1.25rem paddingLeft */
-  nested?: boolean;
   id?: string;
+  title?: string;
 }
 
-function ListItemLink(props: any) {
-  return (
-    <ListItem
-      button={true}
-      component='a'
-      {...props}
-    />
-  );
-}
+export function AltinnAttachment({ attachments, id, title }: IAltinnAttachmentProps) {
+  const { lang, langAsString, selectedLanguage } = useLanguage();
 
-export function AltinnAttachment({ attachments, listDisableVerticalPadding, nested, id }: IAltinnAttachmentProps) {
-  const classes = useStyles();
-  const { selectedLanguage } = useLanguage();
+  const filteredAndSortedAttachments = attachments
+    ?.filter((attachment) => attachment.name)
+    .sort((a, b) => (a.name && b.name ? a.name.localeCompare(b.name, selectedLanguage) : 0));
```

Comment:
`localeCompare` won't sort by numbers if not specifying the `{ numeric: true }` option. So `"1" <  "2"` but `"2" !< "10"`. I'm not sure if that's relevant in the case of attachmentlist, but you could imagine a user uploading files that start with numeric values :sweat_smile:

Reply:
Aha :D Thx, will check it out.